tests/integrations/litestar/test_litestar.py

```diff
@@ -0,0 +1,326 @@
+import functools
+
+import pytest
+
+from sentry_sdk import capture_message
+from sentry_sdk.integrations.litestar import LitestarIntegration
+
+from typing import Any, Dict
```

Comment:
```suggestion
from typing import Any
```

Please use builtin `dict` instead.